README.md

```diff
@@ -3,6 +3,7 @@
   <br/>
   Offix extends capabilities of Apollo GraphQL providing</br>
   fully featured Offline Workflow and Conflict Resolution.
+[![Discord Chat](https://img.shields.io/discord/625400653321076807)](https://discord.gg/mJ7j84m)
```

Comment:
Hey there thanks for your contribution! Can you please add the badge outside the `</p> closing tag but before the `Features` heading?

Reply:
Sure!